### ISSUE TITLE
"images/atp-gpo-proxy1.png" is not showing

### ISSUE THREAD

**Issue by hyoshioka0128** [MicrosoftDocs/windows-itpro-docs#3969]
https://docs.microsoft.com/ja-jp/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet
![configure-proxy-internet](https://user-images.githubusercontent.com/40815708/59170107-1c618400-8b78-11e9-820f-a7f163edc619.png)

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: 83207fc9-2c91-eb07-4237-6bb1f42e3778
* Version Independent ID: 5db33cf3-2dcb-bb33-a252-3a13ca9737f9
* Content: [コンピューターのプロキシとインターネット接続の設定を構成する](https://docs.microsoft.com/ja-jp/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet)
* Content Source: [windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @mjcaparas
* Microsoft Alias: **macapara**

**Comment by e0i** [comment]
@officedocsbot assign @e0i

**Comment by e0i** [comment]
@officedocsbot assign @dongmicro
@officedocsbot add label localization

**Comment by illfated** [comment]
Looks like a conversion issue in the HTML generating process. The first image is clearly not linked correctly when you view the HTML source:

The correct HTML tag should be
`<img src="images/atp-gpo-proxy1.png" alt="Image of Group Policy setting" data-linktype="relative-path">` 
-- but instead, the HTML tag `<a href="images/atp-gpo-proxy1.png" data-linktype="relative-path">`
has been generated, which will not display any image.
```html
<li>管理用テンプレート &gt; Windows コンポーネント &gt; データの収集とプレビュービルド &gt; 接続されているユーザーエクスペリエンスとテレメトリサービスの認証済みプロキシの使用を構成します。<ul>
--
  | <li><strong>有効</strong>に設定して、<a href="images/atp-gpo-proxy1.png" data-linktype="relative-path">認証された![<strong>プロキシ使用を無効にする</strong>] を選択します。グループポリシー設定のイメージ</a></li>
  | </ul>
  | </li>
  | <li><p><strong>管理用テンプレート &gt; Windows コンポーネント &gt; データの収集とプレビュービルド &gt;、接続されたユーザーエクスペリエンスとテレメトリを構成し</strong>ます。</p>
  | <ul>
  | <li><p>プロキシを構成します。<br>
  | <img src="images/atp-gpo-proxy2.png" alt="グループポリシー設定の画像" data-linktype="relative-path"></p>


```

**Comment by hyoshioka0128** [comment]
en-us:https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet
![configure-proxy-internet_e](https://user-images.githubusercontent.com/40815708/59960898-a1e71b80-950a-11e9-8636-86de0f5b66ac.png)

ja-jp:https://docs.microsoft.com/ja-pj/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet
![configure-proxy-internet_j](https://user-images.githubusercontent.com/40815708/59960899-a90e2980-950a-11e9-9eb0-8a3d3913b31d.png)

**Comment by e0i** [comment]
@officedocsbot assign @MollyMalone

**Comment by e0i** [comment]
@officedocsbot unassign @MollyMalone
@officedocsbot unassign @TinaMcN

**Comment by e0i** [comment]
@officedocsbot unassign @dongmicro

**Comment by illfated** [comment]
@e0i & @TinaMcN : I got an idea from looking at the Github source text to see the difference between the 2 images:
```html
- Administrative Templates > Windows Components > Data Collection and Preview Builds > Configure Authenticated Proxy usage for the Connected User Experience and Telemetry Service
  - Set it to **Enabled** and select **Disable Authenticated Proxy usage**:
  ![Image of Group Policy setting](images/atp-gpo-proxy1.png)
- **Administrative Templates > Windows Components > Data Collection and Preview Builds > Configure connected user experiences and telemetry**:
  - Configure the proxy:<br>
    ![Image of Group Policy setting](images/atp-gpo-proxy2.png)
```
Notice that the second image, which gets converted correctly on the ja-jp page, is preceded by a `<br>` HTML "break" tag, whereas the previous image is not.

Both images are placed similarly (possibly identically) on the en-us page, so I presume it should not cause any change to the en-us version of the page to add a `<br>` HTML "break" tag to the line before the first image (which caused this issue). Would it be useful if I propose a PR to have that extra `<br>` HTML "break" tag added, something like this? (see below)
```html
- Administrative Templates > Windows Components > Data Collection and Preview Builds > Configure Authenticated Proxy usage for the Connected User Experience and Telemetry Service
  - Set it to **Enabled** and select **Disable Authenticated Proxy usage**: <br>
  ![Image of Group Policy setting](images/atp-gpo-proxy1.png)
- **Administrative Templates > Windows Components > Data Collection and Preview Builds > Configure connected user experiences and telemetry**:
  - Configure the proxy:<br>
    ![Image of Group Policy setting](images/atp-gpo-proxy2.png)
```

**Comment by TinaMcN** [comment]
Hi @illfated thanks for looking into this.  I checked other languages (Chinese, French) etc. and they are all displaying the image correctly so it seems to be an issue with Japanese only.  Therefore adding the <br> tag probably won't help.   I will convert the Japanese topic to human translation and it should be completed by 4-Nov.

**Comment by TinaMcN** [comment]
@hyoshioka0128 記事は現在、人間が翻訳されています。 問題が解決したことを確認してください。 フィードバックをありがとう。
(The article is now human translated.  Please verify that the issue is fixed.  Thanks again for the feedback.)

**Comment by illfated** [comment]
The en-us page (https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet) was last updated 11/07/2019 (November 7, 2019) -- at the time of this comment. This change was based on commit https://github.com/MicrosoftDocs/windows-itpro-docs/commit/35af62b13d0272bdeabd9ceca23d57e97598f59d from November 1, 2019.

The ja-jp page (https://docs.microsoft.com/ja-jp/windows/security/threat-protection/microsoft-defender-atp/configure-proxy-internet) was updated 2019/11/12 (2019, November 12), and it shows both images.

This issue looks very much resolved to me.

**Comment by hyoshioka0128** [comment]
@TinaMcN @illfated - Fixed.
Closed #3969.
